taxi/urls.py

```diff
@@ -0,0 +1,10 @@
+from django.urls import path
```

Comment:
Add content to `static/css/styles.css` to display that static file connected properly.